README.md

```diff
@@ -1,2 +1,25 @@
-# nva-institution-api
-API for administration of Institutions that are Customers in NVA
+# NVA Customer API
+API for administration of Institutions that are Customers in NVA.
+
+## Deployment considerations
+
+Read this before deploying this template.
+
+### Parameter overrides
+
+Ensure all parameters defined in the Parameter section of [SAM template](template.yaml#L26) are set.
+
+### Resources outside SAM template
+
+These resources are defined outside the SAM template, but used in the template. See details below.
+
+#### Customers DynamoDB Table
+
+Defined by template in 
+[cloudformation\dynamodb_customer_table.yml](cloudformation\dynamodb_customer_table.yml) and passed 
+to SAM template as parameters with table name and index names. Ensure table and indices is up to 
+date before deploying new SAM template.
```

Comment:
This is good enough for now, but ideally we want these dynamodb changes to be defined in `template.yaml` as for the rest of the service when we know how to handle our migrations on persistent databases with cloudformation.